01week/resume/css/style.css

```diff
@@ -60,11 +60,3 @@ h5{
 p {
   text-align: center;
 }
-
-.page {
-  text-align=center
```

Comment:
Erroneous use of the = sign here. Colons are the only way this works, my mistake.